WMFData/Sources/WMFData/Data Controllers/YearInReview/YearInReviewDataController.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+final public class YearInReviewDataController {
```

Comment:
@mazevedofs I actually do want this empty class here, just in case we need to hook into it soon, thanks. Can you rename to WMFYearInReviewDataController?